modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/paths/EVPOrganization.path

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+<title>EVPOrganization</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">EVPOrganization</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>ASSERT_STATUS</td>
+	<td>//p[normalize-space()='${key_organizationName}']//following::p[normalize-space()='${key_email}']//following::p[normalize-space()='${key_status}']</td>
+	<td></td>
+</tr>
```

Comment:
Hi @joelrocha68, it isn't a good idea to use style elements in our paths (p, h1, h2, etc...), because it makes our paths fragile and easier to break due to front-end changes. If you use `*`, will be better, because it will deal with any style.